.github/workflows/pod-operation.yml

```diff
@@ -0,0 +1,37 @@
+name: "Pod Operation Workflow"
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  check-pod-operation:
+    name: Check Pod Operation
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./pod-operation
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
```

Comment:
Issue: the actions-rs/toolchain action hasn't been maintained in a while (last commit four years ago), and we get deprecation notices about Node.js 16 and the `set-output` command. One alternative seems to be [dtolnay/rust-toolchain](https://github.com/dtolnay/rust-toolchain) as mentioned in actions-rs/toolchain#216.